Frameworks/CoreText/DWriteWrapper_CTFont.mm

```diff
@@ -358,5 +360,41 @@ CFStringRef _DWriteFontCopyName(const ComPtr<IDWriteFontFace>& fontFace, CFStrin
         informationalStringId = DWRITE_INFORMATIONAL_STRING_NONE;
     }
 
-    return _DWriteFontCopyInformationalString(fontFace, informationalStringId);
+    return _DWriteFontCopyInformationalString(fontFace, informationalStringId, actualLanguage);
+}
+
+static constexpr CTFontTableTag sc_tags[] = { kCTFontTableBASE, kCTFontTableCFF,  kCTFontTableDSIG, kCTFontTableEBDT, kCTFontTableEBLC,
+                                              kCTFontTableEBSC, kCTFontTableGDEF, kCTFontTableGPOS, kCTFontTableGSUB, kCTFontTableJSTF,
+                                              kCTFontTableLTSH, kCTFontTableOS2,  kCTFontTablePCLT, kCTFontTableVDMX, kCTFontTableVORG,
+                                              kCTFontTableZapf, kCTFontTableAcnt, kCTFontTableAvar, kCTFontTableBdat, kCTFontTableBhed,
+                                              kCTFontTableBloc, kCTFontTableBsln, kCTFontTableCmap, kCTFontTableCvar, kCTFontTableCvt,
+                                              kCTFontTableFdsc, kCTFontTableFeat, kCTFontTableFmtx, kCTFontTableFpgm, kCTFontTableFvar,
+                                              kCTFontTableGasp, kCTFontTableGlyf, kCTFontTableGvar, kCTFontTableHdmx, kCTFontTableHead,
+                                              kCTFontTableHhea, kCTFontTableHmtx, kCTFontTableHsty, kCTFontTableJust, kCTFontTableKern,
+                                              kCTFontTableKerx, kCTFontTableLcar, kCTFontTableLoca, kCTFontTableMaxp, kCTFontTableMort,
+                                              kCTFontTableMorx, kCTFontTableName, kCTFontTableOpbd, kCTFontTablePost, kCTFontTablePrep,
+                                              kCTFontTableProp, kCTFontTableSbit, kCTFontTableSbix, kCTFontTableTrak, kCTFontTableVhea,
+                                              kCTFontTableVmtx };
+
+CFArrayRef _DWriteCopyAvailableFontTables(const ComPtr<IDWriteFontFace>& fontFace) {
```

Comment:
nit: prefer taking ComPtr values by unboxed ptr

I know there's precedent for doing the wrong thing in this code, so it may not be worth fixing.